index.html

```diff
@@ -330,40 +339,25 @@ <h3 class="languages">Languages</h3>
             <div class="event17">
               <h3>Frameworks</h3>
               <button type="submit" class="dropdown">
-                <img src="./images/Icon-Chevron-Right-Larger.png" alt="" />
+                <img src="./images/Disabled.png" alt="" />
               </button>
             </div>
             <img src="./images/Separator-Bottom.png" alt="" class="separator" />
             <div class="event18">
               <h3>Skills</h3>
               <button type="submit" class="dropdown">
-                <img src="./images/Icon-Chevron-Right-Larger.png" alt="" />
+                <img src="./images/Disabled.png" alt="" />
               </button>
             </div>
             <img src="./images/Separator-Bottom.png" alt="" class="separator" />
```

Comment:
Kindly check the image paths, the necessary icons are not here, You can check the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) for more understanding.
**Your design**
![arrow not](https://user-images.githubusercontent.com/73854999/199366776-41901d9a-3381-46fc-a86c-1a0c8e3cc69c.jpg)

**Figma**
![fig](https://user-images.githubusercontent.com/73854999/199366885-9de8b2ea-2b06-467d-b81f-5a6b8dea81bf.jpg)

Reply:
resolved to meet figma design